tests/Cms/App/AppTranslationsTest.php

```diff
@@ -114,6 +114,12 @@ public function testTranslationFromCurrentLanguage()
                     'translations' => [
                         'button' => 'Knopf'
                     ]
+                ],
+                [
+                    'code'         => 'en',
+                    'translations' => [
+                        'hello' => 'Hello'
+                    ]
```

Comment:
@bastianallgeier This is for notice error. You can try`with rollback `src/Cms/AppTranslations.php` update in this PR.

Reply:
Tests for: `EN` translations exists but `EN` language translations not exists.

Reply:
Ah, nice! I missed that.